### ISSUE TITLE
Conflicting Dependencies

### ISSUE THREAD

**Issue by MamoudouSD** [googleapis/google-cloud-python#12442]
I'm trying to solve a dependency problem in the training design of an AI model (RNN). 
Issue summary:

`
ERROR: ResolutionImpossible: for help visit https://pip.pypa.io/en/latest/user_guide/#fixing-conflicting-dependencies

ERROR: Cannot install aiplatform-custom-trainer-script and aiplatform-custom-trainer-script==0.1 because these package versions have conflicting dependencies.`

My configurations are : 

`
# Resources
TRAIN_COMPUTE = 'n1-standard-4'
DEPLOY_COMPUTE = 'n1-standard-4'
TRAIN_IMAGE = 'us-docker.pkg.dev/vertex-ai/training/tf-cpu.2-5:latest'
DEPLOY_IMAGE ='us-docker.pkg.dev/vertex-ai/prediction/tf2-cpu.2-5:latest'

experiment = '01a'
CMDARGS = [
    "--epochs=" + str(epochs),
    "--batch_size=" + str(batch_size),
    "--var_target=" + var_target,
    "--project_id=" + project_id,
    "--bq_project=" + bq_project,
    "--bq_dataset=" + bq_dataset,
    "--bq_table_train=" + bq_table_train,
    "--bq_table_test=" + bq_table_test,
    "--region=" + region,
    "--experiment=" + experiment,
    "--series=" + series,
    "--experiment_name=" + EXPERIMENT_NAME,
    "--run_name=" + RUN_NAME
]

customJob = aiplatform.CustomJob.from_local_script(
    display_name = f'{series}_{experiment}_{TIMESTAMP}',
    script_path = file,
    container_uri = TRAIN_IMAGE,
    args = CMDARGS,
    requirements = ['tensorflow_io', f'google-cloud-aiplatform==1.34.0', 'db-dtypes', f"protobuf==3.19.0"],
    replica_count = 1,
    machine_type = TRAIN_COMPUTE,
    accelerator_count = 0,
    base_output_dir = f"{URI}/models/{TIMESTAMP}",
    staging_bucket = f"{URI}/models/{TIMESTAMP}",
    labels = {'series' : f'{series}', 'experiment' : f'{experiment}', 'experiment_name' : f'{EXPERIMENT_NAME}', 'run_name' : f'{RUN_NAME}'}
)

customJob.run(
    service_account = SERVICE_ACCOUNT,
    tensorboard = tb.resource_name
)
`

I don't know why the error and how to fix it. Please I need Help
